workflows/ci-simple/versions/0.0.1/docs/ci-simple.md

```diff
@@ -0,0 +1,87 @@
+# ci simple
+
+## Summary
+                 
+This CI pipeline builds a docker image using Kaniko, uploads image metadata to CSDP, and tests the image
+https://codefresh.io/csdp-docs/docs/getting-started/quick-start/create-ci-pipeline/
+
+
+## Inputs/Outputs
+
+#### Parameters
+* GIT_REPO_URL (required) - The repository you want to clone (for example: https://github.com/codefresh-io/cli-v2)
+* IMAGE_NAME (required) - The image name to give to the built image
+* IMAGE_TAG (optional) - The tag that will be given to the image. defaults to `latest`
+* GIT_REVISION (optional) - The revision to checkout. defaults to `main`
+* GIT_BRANCH (optional) - The git branch for reporting the image. defaults to `main`
+* GIT_COMMIT_URL (optional) - The git commit url. defaults to `''`
+* GIT_COMMIT_MESSAGE (optional) - The git commit message. defaults to `''`
+* DOCKERFILE (optional) - The path to your dockerfile. defaults to `Dockerfile`
+* GIT_TOKEN_SECRET (required) - The k8s secret name that contains a key named `token` with the git secret inside it. defaults secret name `github-token` . https://codefresh.io/csdp-docs/docs/getting-started/quick-start/create-ci-pipeline/#create-a-personal-access-token-pat
+* CONTEXT (optional) - The context of file system that will be passed. defaults to `.`
+* REGISTRY_CREDS (required) - The Kubernetes secret with the standard registry username, password and domain. defaults secret name `registry-creds` . https://codefresh.io/csdp-docs/docs/getting-started/quick-start/create-ci-pipeline/#create-registry-creds-secret
+* DOCKER_CONFIG (required) - The k8s secret name from type docker-registry with all registries credentials you need to pull from or push to. defaults secret name `docker-config` . https://codefresh.io/csdp-docs/docs/getting-started/quick-start/create-ci-pipeline/#create-docker-registry-secret
```

Comment:
I think it would be clearer from the Create Pipeline wizard if all the inputs that are secrets end with _SECRET. So DOCKER_CONFIG_SECRET and REGISTRY_CREDS_SECRET.

Reply:
done